lib/v2/gov/moe/moe.js

```diff
@@ -45,27 +45,33 @@ module.exports = async (ctx) => {
                     const firstA = item.find('a');
                     const itemUrl = new URL(firstA.attr('href'), moeUrl).href;
 
-                    const infos = await ctx.cache.tryGet(itemUrl, async () => {
-                        const res = {};
-                        const response = await got(itemUrl);
-                        const data = cheerio.load(response.data);
+                    let infos
 
-                        if (itemUrl.indexOf('www.gov.cn') > -1) {
-                            res.description = data('div.pages_content > p').html();
-                            res.title = data('div.article > h1').text();
-                        } else if (itemUrl.indexOf('srcsite') > -1) {
-                            res.description = data('div#xxgk_content_div').html();
-                            res.title = data('div#xxgk_content_redheadbg > h1').text();
-                        } else if (itemUrl.indexOf('jyb_') > -1) {
-                            res.description = data('div.TRS_Editor').html();
-                            res.title = data('div.TRS_Editor').siblings('h1').first().text();
+                    // some live pages have no content, just return the liva page url
+                    if(itemUrl.indexOf('/live/') > -1) {
+                        infos = {
+                            description: firstA.html()
                         }
```

Comment:
```suggestion
                        };
```
Lint your code before pushing pls.